test/gc/binding.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+try {
+  module.exports = require('./build/Release/binding').ongc;
+} catch (e) {
+  module.exports = require('./build/Debug/binding').ongc;
```

Comment:
You're right that the build won't compile a debug version but it seems convenient for manual testing/debugging.  What would you do instead?